packaging/windows/bundle.wxs

```diff
@@ -32,6 +32,13 @@
     <util:DirectorySearch Path="[PreviousInstallFolder]" Variable="DOTNETHOME" After="PreviousInstallFolderSearch" Condition="PreviousInstallFolder" />
 
     <Chain DisableSystemRestore="yes" ParallelCache="yes">
+      <ExePackage SourceFile="$(var.CrtRedistributableSourcePath)"
+                  DetectCondition='(VCRedist_KeyExists AND VCRedist_KeyValue &gt;= "v14.0.23506.00")'
+                  InstallCommand="/q /norestart"
+                  RepairCommand="/q /norestart /repair"
```

Comment:
I wouldn't add a `RepairCommand`. At first glance, it looks like the right thing to do but what it really means is that repairing your product means repairing the dependency that is really an entity all to its own. If the VC Redist is broken, better to let the user repair it directly.